.github/aggregate.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs')
+const process = require('process')
+const path = require('path')
+
+// params: list of json reports to aggregate
+const inputs = process.argv.slice(2)
+
+const processReport = (filePath) => {
+  const resolvedPath = path.isAbsolute(filePath) ? filePath : path.resolve(process.cwd(), filePath);
+  const json = JSON.parse(fs.readFileSync(resolvedPath, 'utf8'))
+
+  // extract the TestMetadata
+  const { TestMetadata, ...rest } = json
+  const { time, meta } = TestMetadata || {}
+  const { version, job_url, gateway_url } = meta || {}
```

Comment:
I'll have to add types and a schema validation library (like zod) since the whole thing is (side effect) -> ETL -> (side effect).

To me that means we:
- add transpilation, create a module, package.json, etc,
- add types + schemas + validation of I/Os

I would keep it simple, but that's up to you, let me know!